omnibus/config/projects/agent.rb

```diff
@@ -110,32 +109,23 @@
   dependency 'curl'
 end
 
-dependency 'cacerts'
 # creates required build directories
 dependency 'datadog-agent-prepare'
 
 # Datadog agent
 dependency 'datadog-agent'
 
 # Additional software
-dependency 'pip'
-dependency 'datadog-agent-integrations'
+dependency 'cacerts'
 dependency 'datadog-a7'
-dependency 'datadog-agent-env-check'
-dependency 'jmxfetch'
-
-# External agents
-dependency 'datadog-process-agent' # Includes network-tracer
-
+dependency 'datadog-agent-integrations'
 if osx?
   dependency 'datadog-agent-mac-app'
 end
 
-# Remove pyc/pyo files from package
-# should be built after all the other python-related software defs
-unless osx?
-  dependency 'py-compiled-cleanup'
```

Comment:
same as above, cleanup was moved into `datadog-agent-finalize` and fixed for OSX